### ISSUE TITLE
Prototype Inception-based classifier for eyes closed/open data

### ISSUE THREAD

**Issue by jdpigeon** [NeuroTechX/neurodoro#10]
Following the Tensorflow for Poets tutorial, I've tested the ability of a retrained Inception network to identify open vs closed eye epochs of data. 

The idea is to convert our EEG data into spectrograms (2s epochs) and then feeding these images (as jpgs to start with) to this classifer and teaching it distinguish between open vs closed eye spectrograms.

However, taking a look at the spectrograms we're getting from Neurodoro data, I want to reevaluate this step. There may be errors in the processing code in the jupyter notebook or even in the app's filtering code.

**Comment by jdpigeon** [comment]
Works with ~98% validation accuracy

See inception_classifier notebook
